scalableminds/webknossos
# Panel content and new controls followup

Two simple fixes of issues probably caused by the panel reorganization and new shortcuts or so.

This PR also unifies the naming for the context menu in the frontend. Thus there are some more line changes than necessary.

### URL of deployed dev instance (used for testing):
- https://panelcontentandnewcontrolsfollowup.webknossos.xyz

### Steps to test:
- Open a dataset with a segmentation layer in view mode
- New new keyboard shortcuts mode
- Open the context menu while hovering a segment
- There should be the option to load the matching precomputed mesh (but it is probably not selectable)
- The copy segment id icon should now support dark mode (to test this, switch to dark mode). The icon should then be white. 

### Issues:
- fixes #5611 #5610 #5620 

------
